ports/zeroptianinvasion/port.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "zeroptianinvasion.zip",
+  "items": [
+    "Zeroptian Invasion.sh",
+    "zeroptianinvasion"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Zeroptian Invasion",
+    "porter": [
+      "tabreturn",
+      "Fraxinus88"
+    ],
+    "desc": "Zeroptian Invasion is a classic shoot 'em up with mechanics from other arcades of the 80s. Travel back in time with this unique pixelated experience featuring an incredible soundtrack in chiptune.",
+    "desc_md": null,
+    "inst": "Purchase the game on itch.io (https://josyan.itch.io/zeroptian-invasion) or Steam (https://store.steampowered.com/app/974110/Zeroptian_Invasion). For itch.io, download the ZeroptianInvasion_itchio.zip version, extract it and copy the game files to the port's gamedata folder. For Steam, there's no need to extract any files, just copy them across.",
+    "inst_md": "Purchase the game on [itch.io](https://josyan.itch.io/zeroptian-invasion) or [Steam](https://store.steampowered.com/app/974110/Zeroptian_Invasion). For itch.io, download the *ZeroptianInvasion\\_itchio.zip* version, extract it and copy the game files to the port's *gamedata* folder. For Steam, there's no need to extract any files, just copy them across.",
+    "genres": [
+      "arcade"
+    ],
+    "image": null,
+    "rtr": false,
+    "exp": false,
+    "runtime": null,
+    "reqs": [],
+    "arch": [
+      "armhf"
```

Comment:
Is this a 32 bit port? If yes it also needs 32 bit export in the .sh
else this is wrong.

Reply:
This one is aarch64

```
file gmloadernext
gmloadernext: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (GNU/Linux), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, BuildID[sha1]=7a9e37b2cb8bdbda179b3d48b1c4ed65ba0ebc44, for GNU/Linux 3.7.0, not stripped
```

Reply:
Oops! First gmloader-next port for me. Corrected